requirements.txt

```diff
@@ -8,8 +8,7 @@ psutil >= 5.0
 sortedcontainers !=2.0.0, !=2.0.1
 tblib >= 1.6.0
 toolz >= 0.8.2
-tornado >= 5;python_version<'3.8'
-tornado >= 6.0.3;python_version>='3.8'
+tornado >= 6.0.3
```

Comment:
This would also allow us to make the Conda package `noarch` again, which would help speed up building time in conda-forge